sled-agent/src/storage_manager.rs

```diff
@@ -800,7 +863,12 @@ impl StorageWorker {
         disk: UnparsedDisk,
     ) -> Result<(), Error> {
         info!(self.log, "Deleting disk: {disk:?}");
-
+        // TODO: Don't we need to do some accounting, e.g. for all the information
+        // that's no longer accessible? Or is that up to Nexus to figure out at
+        // a later point-in-time?
+        //
+        // If we're storing zone images on the M.2s for internal services, how
+        // do we reconcile them?
```

Comment:
We definitely needed to do this before this PR, but I'm labeling it. This will be more and more relevant as we move configs from the ramdisks onto U.2s.